src/services/helpdesk/hooks/index.js

```diff
@@ -14,9 +15,28 @@ Deine ${data.cloud}
 	`;
 }
 
+async function generateSystemInformation(account) {
+	const { userId, username } = account
+
+	const userInformation = await userModel.userModel.findOne(userId)
+		.select({ 'roles': '1', 'email': '1' })
+		.populate('roles', 'name')
+		.lean()
+
+	const roles = userInformation.roles.length ? userInformation.roles.map(info => info.name) : 'NO ROLES'
+	const email = userInformation.email || 'NO EMAIL'
+	const systemInformation =
```

Comment:
Codacy found an issue: [There should be no line break before or after '='.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5822205)